clients/spark-extensions/pom.xml

```diff
@@ -136,10 +148,10 @@
       <plugin>
         <groupId>org.projectnessie</groupId>
         <artifactId>nessie-apprunner-maven-plugin</artifactId>
-        <version>${project.version}</version>
+        <version>${iceberg.nessie.version}</version>
```

Comment:
Unnecessary change - the plugin can still be the project's version

Reply:
it's necessary with the `nessie-model` changes and startup breaks